src/components/Button.js

```diff
@@ -3,13 +3,18 @@ import PropTypes from 'prop-types';
 
 function Button(props) {
   // eslint-disable-next-line react/prop-types
```

Comment:
Do not disable react/prop-types rule. Please remove this line and add the necessary rule to clear this warning/error.